onadata/apps/logger/models/instance.py

```diff
@@ -388,6 +385,12 @@ def get_full_dict(self, load_existing=True):
             if not self.date_created:
                 self.date_created = submission_time()
 
+            if not self.date_modified:
+                doc[DATE_MODIFIED] = self.date_created
```

Comment:
Should this be getting, the date format treatment as below or is it handled elsewhere?
```python
doc[DATE_MODIFIED] = self.date_created.strftime(MONGO_STRFTIME)
```